flow-typed/types/media-info.js

```diff
@@ -12,6 +12,7 @@ declare type OTTProviderMediaInfoObject = OVPProviderMediaInfoObject & {
   fileIds?: string,
   streamerType?: string,
   urlType?: string,
+  adapterData?: Array<Object>,
```

Comment:
```suggestion
  adapterData?: Array<{[key: string]: string}>,
```

Reply:
as this type repeats itself than better define a type and use in both places

Reply:
I'm not sure it's typed as value, objectType.